useful-scripts/misc-test-utils/src/services/subjectService.ts

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-console */
+import axios from 'axios';
+import globby from 'globby';
+import path from 'path';
+import { v4 } from 'uuid';
+import fs from 'fs';
+import FormData from 'form-data';
+import chalk from 'chalk';
+import { projectRoot } from '../config';
+import errorHandler from '../utils/errorHandler';
+import { SubjectData } from '../types/SubjectData';
+import sleep from '../utils/sleep';
+
+const cwd = projectRoot;
+
+const { ADMIN_URL, JWT_TOKEN } = process.env;
+
+const subjectService = {
+  addSubject: async (releaseId: string): Promise<string | null> => {
+    const finalZipFileGlob = await globby(
+      `${cwd}/zip-files/clean-test-zip-*.zip`,
+    );
+    const oldPath = finalZipFileGlob[0];
+    const newPath = path.resolve(oldPath, '..', path.basename(oldPath));
+    fs.renameSync(oldPath, newPath);
+    const form = new FormData();
+    form.append('zipFile', fs.createReadStream(newPath));
+    try {
+      const res = await axios({
+        maxContentLength: Infinity,
+        maxBodyLength: Infinity,
+        method: 'POST',
+        url: `${ADMIN_URL}/api/release/${releaseId}/zip-data?title=importer-subject-${v4()}`,
+        data: form,
+        headers: {
+          ...form.getHeaders(),
+          Authorization: `Bearer ${JWT_TOKEN}`,
```

Comment:
Similarly to `publicationService`, we can refactor all of these Axios invocations with a pre-configured Axios instance.